admin/partials/settings/auto-backup.php

```diff
@@ -17,37 +17,30 @@
 
 ob_start();
 ?>
+<h1>
+<?php
+esc_html_e( 'Auto Updates', 'boldgrid-backup' );
+?>
+</h1>
+<p>
+	WordPress has the ability to auto update both itself and your plugins and themes. Keeping your software up to date is very important, and this automation helps you more easily do that.
```

Comment:
The text on this page needs to be translatable.

Reply:
resolved